WordPress/running_post_startup_scripts.md

```diff
@@ -24,22 +24,26 @@ A custom startup script has many use cases. The following are some scenarios for
 
 Configurations for nginx are defined in **/etc/nginx/conf.d/default.conf** and **/etc/nginx/conf.d/spec-settings.conf** files. You can update any of these files using a startup script. For instance, the following commands in startup script changes the nginx configuration for max simultaneous connections to 20000.
 
-``` 
+``` bash
```

Comment:
@rabollin all code blocks should specify the target language, this help render rich formatting

Reply:
taken care of.